examples/account/account_create_transaction_create_with_alias.py

```
@@ -79,100 +84,82 @@ def generate_main_and_alias_keys() -> tuple[PrivateKey, PrivateKey]:
     response = transaction.execute(client)
-    new_account_id = response.account_id
 
+    # Safe retrieval of account ID
+    new_account_id = response.account_id
     if new_account_id is None:
-        raise RuntimeError(
-            "AccountID not found in receipt. Account may not have been created."
-        )
+        try:
+            new_account_id = response.get_receipt(client).account_id
+        except Exception:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)